docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -16,7 +16,25 @@ Broadly speaking, there are three kinds of ScPCA data you might wish to work wit
     - These data are used for automated testing, but you can also use them while developing your module if smaller files helps make development more efficient.
     - You do not need an AWS account to download the test files, so you can use this data before you are granted full access to ScPCA data.
 
-## Accessing ScPCA Data
+
+!!! tip "Log into AWS CLI before running the download scripts"
+    Below, we describe how to use both the `data-download.py` and `results-download.py` script to download ScPCA data and result files, respectively.
+    Before running either of these scripts locally (e.g., not from a [virtual computer](../../aws/index.md#lightsail-for-research-virtual-computing-with-aws)), you will need to be [logged into your AWS CLI profile](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
```

Comment:
I want to clarify that not all virtual computers don't require login; they might have their own virtual systems! So maybe we want to say lightsail specifically as well...

We also probably should say somewhere that downloading test data doesn't require logging in. 

```suggestion
    Before running either of these scripts locally (i.e., not from an ALSF-provided [virtual computer](../../aws/index.md#lightsail-for-research-virtual-computing-with-aws)), you will need to be [logged into your AWS CLI profile](../../technical-setup/environment-setup/configure-aws-cli.md#logging-in-to-a-new-session).
```